### PR TITLE
Removed all the tests without tier and updated the docstring.

### PR DESCRIPTION
**Pull request by san7ket**
#6291

### CODE REVIEW

**Comment by oshtaier** [comment]
huh, it is sad to remove so much coverage, but if everyone okay - I am fine too

**Comment by oshtaier** [comment]
@omaciel we need your magic here =)

**Comment by san7ket** [comment]
Conflicts resolved.

File: tests/foreman/ui_airgun/test_smartvariable.py [hunk 1]
```diff
@@ -3,7 +3,7 @@
 
 :Requirement: Variables
 
-:CaseAutomation: Automated
+:CaseAutomation: notautomated
```
Comment (abalakh) [inline tests/foreman/ui_airgun/test_smartvariable.py]: Not automated?

File: tests/foreman/ui_airgun/test_smartvariable.py [hunk 2]
```diff
@@ -19,9 +19,11 @@
 from nailgun import entities
 
 from robottelo.api.utils import publish_puppet_module
-from robottelo.constants import CUSTOM_PUPPET_REPO, DEFAULT_LOC_ID, ENVIRONMENT
+from robottelo.constants import DEFAULT_LOC_ID
 from robottelo.datafactory import gen_string
-from robottelo.decorators import fixture, tier2, upgrade
+from robottelo.decorators import tier2, upgrade
+from robottelo.constants import CUSTOM_PUPPET_REPO, ENVIRONMENT
```
Comment (abalakh) [inline tests/foreman/ui_airgun/test_smartvariable.py]: Looks like conflicts resolution - 2 separate imports from constants, pls fix

File: tests/foreman/ui_airgun/test_smartvariable.py [hunk 3]
```diff
@@ -19,9 +19,11 @@
...
+from robottelo.decorators import fixture
```
Comment (abalakh) [inline tests/foreman/ui_airgun/test_smartvariable.py]: same with decorators

**Pull request merged**